src/core_modules/capture-core/components/WidgetEnrollmentEventNew/Validated/useBuildNewEventPayload.ts

```diff
@@ -116,8 +115,8 @@ export const useBuildNewEventPayload = ({
     };
 
     const buildNewEventPayload = (
-        saveType: ?$Values<typeof addEventSaveTypes>,
-        relatedStageRef?: {| current: (?RelatedStageRefPayload) |},
+        saveType?: keyof typeof addEventSaveTypes,
```

Comment:
Change for `saveType?: typeof addEventSaveTypes[keyof typeof addEventSaveTypes]`, this is the pattern how `$Values<typeof T>` should be converted